topology/graph/message.go

```diff
@@ -79,6 +79,10 @@ func UnmarshalWSMessage(msg shttp.WSMessage) (string, interface{}, error) {
 			}
 			context.TimeSlice = common.NewTimeSlice(i, i)
 		}
+		switch v := m["topologyFilter"].(type) {
```

Comment:
The issue with the current approach is that we only filter on Sync message if a node appears after a sync it will appear on the WebUI even if not matching the filter.

We should add something in the forwarding message mechanism checking if the node belong to the filter. if we use the subgraph step it will be easy (I hope)

1. get the new node
2. get the subgraph 
  G....SubGraph()
3 check if the node is present
  G...SubGraph().V(<ID of node>)
4 if present let forward the message